habitat-baselines/habitat_baselines/rl/hrl/skills/nn_skill.py

```diff
@@ -154,7 +154,7 @@ def from_config(
             OrderedDict(
                 [
                     (k, action_space[k])
-                    for k in policy_cfg.habitat.task.possible_actions
+                    for k in policy_cfg.habitat.task.actions.keys()
                 ]
```

Comment:
```suggestion
                (
                    (k, action_space[k])
                    for k in policy_cfg.habitat.task.actions.keys()
                )
``` 
Pretty sure this can be a generator comprehension